Project-Euler/Project-Euler/Program.cs

```diff
@@ -20,19 +37,61 @@ public static void PrintIntro()
             //Add a couple more lines, maybe?
         }
 
-        public static void PrintMenuOptions() //This method will be private once I write the method that will call it. blah blah
+        public static int GetMenuSelection()
+        {
+            PrintMenuOptions();
+
+            while (true)
+            {
+                string userSelection = Console.ReadLine().ToUpper();
+
+                if (!string.IsNullOrEmpty(userSelection) && userSelection.Length == 1)
```

Comment:
I would use _string.IsNullOrWhiteSpace_ as that will check for

- Null
- Empty
- and Whitespace